projects/skia/Dockerfile

```diff
@@ -80,38 +80,8 @@ COPY build.sh $SRC/
 COPY skia.diff $SRC/skia/skia.diff
 RUN git apply skia.diff
 
-COPY region_deserialize.options $SRC/skia/region_deserialize.options
-COPY region_set_path.options $SRC/skia/region_set_path.options
 COPY image_filter_deserialize.options $SRC/skia/image_filter_deserialize.options
```

Comment:
@kjlubick - do these two image_filter_deserialize*.options have to stay ?

Reply:
I vaguely remember us experimenting with two different approaches to fuzzing image_filter_deserialize. Looking at the fuzzer history, it seems the different options *do* have an impact on performance. I think it's worth keeping them in for now until we can maybe do a little more investigation.

Reply:
All the options files are deleted except for image_filter_deserialize_width.options which has use_counter=0